### PR TITLE
@Receiver support for @EView and @EViewGroup

### PR DESCRIPTION
**Pull request by simopete**
Implements #1656

### CODE REVIEW

**Comment by simopete** [comment]
Wiki updates: branch 1656_EView_EViewGroup_Receiver in my wiki repo.

File: AndroidAnnotations/androidannotations-core/androidannotations-test/src/main/res/layout/activity_eview_eviewgroup_receiver.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            android:id="@+id/viewWithReceiver"
+            android:layout_width="fill_parent"
+            android:layout_height="wrap_content"/>
+
+    <org.androidannotations.test.receiver.ViewGroupWithReceiver_
+            android:id="@+id/viewGroupWithReceiver"
+            android:layout_width="fill_parent"
+            android:layout_height="wrap_content"/>
+
+</LinearLayout>
```
Comment (dodgex) [inline AndroidAnnotations/androidannotations-core/androidannotations-test/src/main/res/layout/activity_eview_eviewgroup_receiver.xml]: missing new line

File: AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/helper/ValidatorHelper.java [hunk 2]
```diff
@@ -168,8 +170,8 @@ public void enclosingElementHasEActivityOrEFragment(Element element, ElementVali
 		enclosingElementHasOneOfAnnotations(element, validAnnotations, valid);
 	}
 
-	public void enclosingElementHasEActivityOrEFragmentOrEServiceOrEIntentService(Element element, ElementValidation valid) {
-		List<Class<? extends Annotation>> validAnnotations = asList(EActivity.class, EFragment.class, EService.class, EIntentService.class);
+	public void enclosingElementHasEActivityOrEFragmentOrEServiceOrEIntentServiceOrEViewOrEViewGroup(Element element, ElementValidation valid) {
```
Comment (dodgex) [inline AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/helper/ValidatorHelper.java]: have you checked that this method is not used somehwere else to not break validation in other processors?

Comment (dodgex) [inline AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/helper/ValidatorHelper.java]: okay... just realized that all places where this is called are necessarily part of this PR and that it is only used in one place. ignore this comment :D

**Comment by dodgex** [comment]
for now i did just a fast scroll over review. i'll try to check more detailed this evening.

@WonderCsabo would be nice if you try to review this one too. this would be an awesome addition to the 4.1 release :)

**Pull request merged**